ivy_tests/test_ivy/helpers.py

```diff
@@ -980,6 +980,7 @@ def test_function(
         if not max(container_flags) and ivy.native_inplace_support:
             # these backends do not always support native inplace updates
             assert ret.data is out.data
+    ivy.unset_backend()
```

Comment:
this doesn't look right, why are we unsetting the backend here, where has it been set? It looks like the backend at this point will be the backend we are testing for, which should not be unset.